File: brad-kerr/tip-calculator.js
function calculateTip(amount, rating) {
switch (rating.toLowerCase()) {
    case "terrible": 
    return 0
    break;
     case "poor": 
    return Math.ceil(amount*0.05)
    break;
     case "good": 
    return Math.ceil(amount*0.1)
    break;
     case "great": 
    return Math.ceil(amount*0.15)
    break;
     case "excellent":
    return Math.ceil(amount*0.2)
    break;
    default: 
    return "Rating not recognised"
}
}

File: brad-kerr/tip-calculator.py
import math

def calculate_tip(amount, rating):
    rating = rating.lower()
    if rating == "terrible":
        return 0
    elif rating == "poor":
        return math.ceil(amount * 0.05)
    elif rating == "good":
        return math.ceil(amount * 0.1)
    elif rating == "great":
        return math.ceil(amount * 0.15)
    elif rating == "excellent": 
        return math.ceil(amount * 0.2)
    else: 
        return "Rating not recognised"
